test/library/standard/Path/Prachi-Muskaan/testcases_expandUser.prediff

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+
+import os
+
+f= open('testcases_expandUser.good','w')
+f.write(os.path.expanduser('~user/') + '\n')
```

Comment:
Hmm.  It'd be beneficial to use a username that the program knows exists on the system (like the current user).  I'm not sure how to go about doing that myself, what resources can you find that would help you with this?

Reply:
If you find out how to get that information when you're in bash, I can tell you how to send it to Chapel :)